actioncable/lib/action_cable/channel/base.rb

```diff
@@ -153,6 +155,14 @@ def initialize(connection, identifier, params = {})
         subscribe_to_channel
       end
 
+      # This method is called after subscription has been added to the channel.
+      # Send confirmation here to avoid race conditions when client tries to perform actions
+      # right after receiving confirmation.
+      def registered!
```

Comment:
Either we make this sound more like `subscribed` (and remove the !) or make it clearer what it does with more words akin to `defer_subscription_confirmation!`